### ISSUE TITLE
CARS: Continuous Evolution for Efficient Neural Architecture Search

### ISSUE THREAD

**Issue by sile** [sile/readings#197]
URL
-----

https://arxiv.org/abs/1909.04977

Abstract
----------

> Searching techniques in most of existing neural architecture search (NAS) algorithms are mainly dominated by differentiable methods for the efficiency reason. In contrast, we develop an efficient continuous evolutionary approach for searching neural networks. Architectures in the population that share parameters within one SuperNet in the latest generation will be tuned over the training dataset with a few epochs. The searching in the next evolution generation will directly inherit both the SuperNet and the population, which accelerates the optimal network generation. The non-dominated sorting strategy is further applied to preserve only results on the Pareto front for accurately updating the SuperNet. Several neural networks with different model sizes and performances will be produced after the continuous search with only 0.4 GPU days. As a result, our framework provides a series of networks with the number of parameters ranging from 3.7M to 5.1M under mobile settings. These networks surpass those produced by the state-of-the-art methods on the benchmark ImageNet dataset.

Table of Contents
--------------------

- (1) Introduction
- (2) Related Works
  - (2-1) Network Architecture Search
  - (2-2) Multi-objective Network Architecture Search
- (3) Approach
  - (3-1) SuperNet of CARS
  - (3-2) Parameter Optimization
  - (3-3) Architecture Optimization
  - (3-4) Continuous Evolution for CARS
    - Parameter Warmup
    - Architecture Optimization
    - Parameter Optimization
  - (3-5) Search Time Analysis
- (4) Experiments
  - (4-1) Experimental Settings
    - SuperNet Backbones
    - Evolution Details
  - (4-2) Experiments on CIFAR-10
    - Small Model Trap
    - NSGA-III vs. pNSGA-III
    - Search on CIFAR-10
    - Evaluate on CIFAR-10
    - Comparison on Searched Cells
  - (4-3) Evaluate on IISVRC2012
- (5) Conclusion

BibTeX
------

```bibtex
@article{Yang_2020,
   title={CARS: Continuous Evolution for Efficient Neural Architecture Search},
   ISBN={9781728171685},
   url={http://dx.doi.org/10.1109/cvpr42600.2020.00190},
   DOI={10.1109/cvpr42600.2020.00190},
   journal={2020 IEEE/CVF Conference on Computer Vision and Pattern Recognition (CVPR)},
   publisher={IEEE},
   author={Yang, Zhaohui and Wang, Yunhe and Chen, Xinghao and Shi, Boxin and Xu, Chao and Xu, Chunjing and Tian, Qi and Xu, Chang},
   year={2020},
   month={Jun}
}
```

See Also
----------

- GitHub: https://github.com/huawei-noah/CARS
